src/sdk/services/oauthService.ts

```diff
@@ -4,7 +4,7 @@ import runtime from '../RuntimeContext';
 class OAuthService {
   public openPopup = (redirectUri?: string, state?: string) => {
     if (!runtime.application.api) {
-      throw new Error("Can't open auth window for addon which has no api configuration in its manifest");
+      throw new Error('Configure API access for this app before calling authorize().');
```

Comment:
You mean before calling  `sdk.authenticate()` ?
```suggestion
      throw new Error('Configure API access for this app before calling authenticate().');
```